README.md

```diff
@@ -13,20 +13,22 @@ The Orchestrate library provides convenient access to the Orchestrate API from a
 
 ## Compatibility
 
-| SDK versions        | Orchestrate versions           |
-| ------------------- | ------------------------------ |
-| master/HEAD         | Orchestrate v21.10.0 or higher |
-| SDK v5.0.x          | Orchestrate v21.10.0 higher    |
-| SDK v4.1.x          | Orchestrate v21.1.5 or higher  |
-| SDK v4.0.x          | Orchestrate v21.1.0 or higher  |
-| SDK v3.1.1          | Orchestrate v2.4 or higher     |
-| SDK v3.1.x          | Orchestrate v2.3 or higher     |
-| SDK v3.0.x          | Orchestrate v2.3-rcX           |
-| SDK v2.3.x          | Orchestrate v2.1 or higher     |
-| SDK v2.2.x          | Orchestrate v2.1 or higher     |
-| SDK v2.1.x          | Orchestrate v2.1 or higher     |
-| SDK v2.0.x          | Orchestrate v2.0               |
-| SDK v1.0.x or lower | Orchestrate v1.0 or lower      |
+| SDK versions        | Orchestrate versions                  |
+| ------------------- | ------------------------------------- |
+| master/HEAD         | Orchestrate v21.11.0-alpha2 or higher |
+| SDK v5.2.x          | Orchestrate v21.11.0-alpha2 or higher |
+| SDK v5.1.x          | Orchestrate v21.10.0-alpha3 or higher |
+| SDK v5.0.x          | Orchestrate v21.10.0-alpha1 or higher |
```

Comment:
I don't see the point in versioning the alphas here, we will need a PR later to remove that

Reply:
Undone